src/cockpit/channel.py

```diff
@@ -285,6 +289,17 @@ def send_data(self, data: bytes) -> bool:
 
         return self._out_sequence < self._out_window
 
+    def send_text(self, data: str) -> bool:
+        """Send UTF-8 string data and handle book-keeping for flow control.
+
+        Similar to `send_bytes`, but for text data.  The data is sent as UTF-8 encoded bytes.
+        """
+        return self.send_bytes(data.encode("UTF-8"))
```

Comment:
Sometimes you use `.decode()` and `.encode()` with an argument, sometimes without.  `utf-8` is the default value, so I think it always makes sense without.

Reply:
Done as a separate commit throughout the code.